pointers/pointers_test.go

```diff
@@ -134,6 +134,104 @@ func TestNewInt64Ptr(t *testing.T) {
 	}
 }
 
+/*
```

Comment:
If no any reason to keep uncommented code, either uncomment it, or remove please